application/lib/tutorial/two/module/explore_menu.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/semantics.dart';
+
+class ExploreMenuPage extends StatefulWidget {
+  const ExploreMenuPage({super.key});
+
+  @override
+  State<ExploreMenuPage> createState() => _ExploreMenuPageState();
+}
+
+class _ExploreMenuPageState extends State<ExploreMenuPage> {
+  bool _hasSpokenIntro = false; // Whether the intro has been spoken yet
+
```

Comment:
```suggestion
final BorderSide _borderSide = BorderSide(width: 2.0, color: Colors.black);

```

Double check I've actually got the syntax right!